NEWS.md

```diff
@@ -1,3 +1,8 @@
+# fish_passage_bulkley_2022_reporting 0.1.2
```

Comment:
need to update the version number in the index.Rmd file too.

FYI - Thinking of testing this package for updateing this file https://github.com/kapsner/autonewsmd